org/mozilla/jss/crypto/Policy.java

```diff
@@ -0,0 +1,32 @@
+package org.mozilla.jss.crypto;
+
+import java.math.BigInteger;
+
+/**
+ * This class helps JSS callers align with local system cryptographic policy.
+ */
+public class Policy {
+    /**
+     * Minimum RSA key length in bits permitted by local policy.
+     */
+    public static final int RSA_MINIMUM_KEY_SIZE = getRSAMinimumKeySize();
```

Comment:
Is the policy likely to change while the server is running? If so we might want to call the method directly instead of storing it as a constant. Otherwise changing the policy will require restarting all running instances.

Reply:
Per the manual entry for `CRYPTO-POLICIES(7)`: 

> If the system administrator changes the system-wide policy level with the update-crypto-policies(8) command it is advisable to restart the system as the individual back-end libraries read the configuration files usually during their initialization. The changes in the policy level thus take place in most cases only when the applications using the back-end libraries are restarted.

So it is generally fine to leave this as is. However, it could be construed as a breaking change. I'm inclined to make these `public static int` constants (thus dropping the `final`) and document that the way to... knowing circumvent policy is to override the constant of choice.